include/eosio/vm/execution_context.hpp

```diff
@@ -268,6 +268,11 @@ namespace eosio { namespace vm {
          get_operand_stack().eat(0);
       }
 
+// Calling execute() with no `args` (i.e. `execute(host_type,jit_visitor,uint32_t)`) results in a "statement has no effect
+// [-Werror=unused-value]" warning. Dissable for this function.
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-value"
```

Comment:
This is the sort of thing I hope we can avoid. I don't want to have to start sprinkling this sort of thing through out the code base.